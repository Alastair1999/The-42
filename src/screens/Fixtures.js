import React from "react";
import {View,
ScrollView,
Image,
Text,
StyleSheet,
Dimensions,
}from "react-native";
import { TopNav, Layout } from "react-native-rapi-ui";
import { Ionicons } from "@expo/vector-icons";
import MapView, { Callout, Marker } from "react-native-maps";


export default function ({ navigation }) {
    return (
        <Layout>
            <TopNav
                middleContent="Fixtures"
                leftContent={
                    <Ionicons
                      name="chevron-back"
                      size={20}
                    />
                  }
                  leftAction={() => navigation.goBack()}
            />
            <ScrollView>
                <View style={styles.container}>
                    <MapView style={styles.map} 
                    initialRegion ={{
                        latitude: 55.953251,
                        longitude: -3.2,
                        latitudeDelta: 0.1,
                        longitudeDelta: 0.065,
                    }}
                    >
                        <Marker 
                            coordinate={{
                                latitude: 57.15917,
                                longitude: -2.08889,
                            }}
                            pinColor = "red"
                            //image ={require('../../assets/club-badges/aberdeen.png')}
                        >
                            <Callout>
                                <Text>Aberdeen</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 55.8597472,
                                longitude: -3.9597528,
                            }}
                            pinColor = "red"
                            //image ={require('../../assets/club-badges/.png')}
                        >
                            <Callout>
                                <Text>Airdrieonians</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 55.86028,
                                longitude: -4.01139,
                            }}
                            pinColor = "yellow"
                            //image ={require('../../assets/club-badges/albion-rovers.png')}
                        >
                            <Callout>
                                <Text>Albion Rovers</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 56.1166444,
                                longitude: -3.7786417,
                            }}
                            pinColor = "orange"
                            //image ={require('../../assets/club-badges/alloa.png')}
                        >
                            <Callout>
                                <Text>Alloa</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 54.99474,
                                longitude: -3.2613,
                            }}
                            pinColor = "yellow"
                            //image ={require('../../assets/club-badges/annan-athletic.png')}
                        >
                            <Callout>
                                <Text>Annan Athletic</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 56.5522694,
                                longitude: -2.5915194,
                            }}
                            pinColor = "#800000"
                            //image ={require('../../assets/club-badges/arbroath.png')}
                        >
                            <Callout>
                                <Text>Arbroath</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 55.4694,
                                longitude: -4.6200,
                            }}
                            pinColor = "white"
                            //image ={require('../../assets/club-badges/ayr.png')}
                        >
                            <Callout>
                                <Text>Ayr United</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 55.87471,
                                longitude: -3.10333,
                            }}
                            pinColor = "red"
                            //image ={require('../../assets/club-badges/bonnyrigg-rose.png')}
                        >
                            <Callout>
                                <Text>Bonnyrigg Rose</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 55.8497,
                                longitude: -4.2055,
                            }}
                            pinColor = "green"
                            //image ={require('../../assets/club-badges/celtic.png')}
                        >
                            <Callout>
                                <Text>Celtic</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 55.9455,
                                longitude: -4.0376,
                            }}
                            pinColor = "red"
                            //image ={require('../../assets/club-badges/clyde.png')}
                        >
                            <Callout>
                                <Text>Clyde</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 57.11296,
                                longitude: -2.09660,
                            }}
                            pinColor = "blue"
                            //image ={require('../../assets/club-badges/cove-rangers.png')}
                        >
                            <Callout>
                                <Text>Cove Rangers</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 55.9383,
                                longitude: -4.5619,
                            }}
                            pinColor = "yellow"
                            //image ={require('../../assets/club-badges/dumbarton.png')}
                        >
                            <Callout>
                                <Text>Dumbarton</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 56.4747,
                                longitude: -2.9690,
                            }}
                            pinColor = "orange"
                            //image ={require('../../assets/club-badges/dundee-united.png')}
                        >
                            <Callout>
                                <Text>Dundee United</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 56.4750,
                                longitude: -2.9722,
                            }}
                            pinColor = "#1a315a"
                            //image ={require('../../assets/club-badges/dundee.png')}
                        >
                            <Callout>
                                <Text>Dundee</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 56.0752,
                                longitude: -3.4414,
                            }}
                            pinColor = "black"
                            //image ={require('../../assets/club-badges/dunfermline.png')}
                        >
                            <Callout>
                                <Text>Dunfermline Athletic</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 56.1887,
                                longitude: -2.9994,
                            }}
                            pinColor = "yellow"
                            //image ={require('../../assets/club-badges/east-fife.png')}
                        >
                            <Callout>
                                <Text>East Fife</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 57.6518,
                                longitude: -3.3208,
                            }}
                            pinColor = "black"
                            //image ={require('../../assets/club-badges/elgin-city.png')}
                        >
                            <Callout>
                                <Text>Elgin City</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 56.0054,
                                longitude: -3.7526,
                            }}
                            pinColor = "black"
                            //image ={require('../../assets/club-badges/falkirk.png')}
                        >
                            <Callout>
                                <Text>Falkirk</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 56.6522,
                                longitude: -2.8849,
                            }}
                            pinColor = "#ADD8E6"
                            //image ={require('../../assets/club-badges/forfar-athletic.png')}
                        >
                            <Callout>
                                <Text>Forfar Atheltic</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 55.9413278,
                                longitude: -4.7268111,
                            }}
                            pinColor = "white"
                            //image ={require('../../assets/club-badges/greenock-morton.png')}
                        >
                            <Callout>
                                <Text>Greenock Morton</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 55.7821,
                                longitude: -4.0584,
                            }}
                            pinColor = "red"
                            //image ={require('../../assets/club-badges/hamilton.png')}
                        >
                            <Callout>
                                <Text>Hamilton Academical</Text>
                            </Callout>
                        </Marker>


                        <Marker 
                            coordinate={{
                                latitude: 55.93917,
                                longitude: -3.23222,
                            }}
                            pinColor = "#800000"
                            //image ={require('../../assets/club-badges/hearts.png')}
                        >
                            <Callout>
                                <Text>Heart of Midlothian</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 55.96167,
                                longitude: -3.16556,
                            }}
                            pinColor = "green"
                            //image ={require('../../assets/club-badges/hibernian.png')}
                        >
                            <Callout>
                                <Text>Hibernian</Text>
                            </Callout>
                        </Marker>

                        <Marker 
                            coordinate={{
                                latitude: 55.95694,
                                longitude: -3.15861,
                            }}
                            pinColor = "white"
                            //image ={require('../../assets/club-badges/hearts.png')}
                        >
                            <Callout>
                                <Text>FC Edinburgh</Text>
                            </Callout>
                        </Marker>
                    </MapView>
                </View>
            </ScrollView>

            
        </Layout>

    );
}

const styles = StyleSheet.create({
    container: {
      flex: 1,
      backgroundColor: '#fff',
      alignItems: 'center',
      justifyContent: 'center',
    },
    map: {
        //width: 500,
        //height: 200.
      width: Dimensions.get('window').width,
      height: Dimensions.get('window').height,
    },
  });