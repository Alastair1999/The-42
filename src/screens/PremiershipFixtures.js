import React from "react";
import {
    View,
    Text,
    StyleSheet,
    ScrollView,
    SectionList,
    SafeAreaView,
    FlatList,
    StatusBar
} from "react-native";
import { TopNav, Layout } from "react-native-rapi-ui";
import { Ionicons } from "@expo/vector-icons";

const DATA = [
    {
        title:"Round 1 - 30/07 to 31/07",
        data: ["Livingston vs Rangers ", "St Johnstone vs Hibernian ", "Kilmarnock vs Dundee United", "Hearts vs Ross County", "St Mirren vs Motherwell", "Celtic vs Aberdeen"]
    },
    {
        title:"Round 2 - 06/08 to 07/08",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 3 - 30/07 to 31/07",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 4",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 5",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 6",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 7",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 8",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 9",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 10",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 11",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 12",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 13",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 14",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 15",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 16",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 17",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 18",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 19",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 20",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 21",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 22",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 23",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 24",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 25",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 26",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 27",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 28",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 29",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 30",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 31",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },
    {
        title:"Round 32",
        data: ["Aberdeen vs St Mirren ", "Motherwell vs Ross County ", "Rangers vs Kilmarnock", "Ross County vs Celtic", "Hibs vs Hearts", "Dundee Utd vs Livingston"]
    },

];

const Item = ({title, data}) => (
    <View style={styles.item}>
        <Text style={styles.title}>{title}</Text>
    </View>
);


export default function PremiershipFixtures ({navigation}) {
    return (
        <Layout>
            <TopNav
                middleContent="Cinch Premiership Fixtures"
                leftContent={
                    <Ionicons
                      name="chevron-back"
                      size={20}
                    />
                  }
                  leftAction={() => navigation.goBack()}
            />
                <View style={styles.container}>
                    <SectionList
                    sections={DATA}
                    keyExtractor={(item, index) => item + index}
                    renderItem={({item}) => <Item title={item} 
                    />}
                    renderSectionHeader={({section: {title}}) => (
                        <Text style={styles.header}>{title}</Text>
                    )}
                    />
                </View>
        </Layout>
    );
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        paddingTop: StatusBar.currentHeight,
        alignContent: 'center',
        marginHorizontal: 16,
    },

    item: {
        backgroundColor: 'pink',
        marginVertical: 2,
        padding: 5,
    },

    header: {
        fontSize: 26,
        backgroundColor: 'white',
        //padding: 15,
        marginVertical: 10,
    },

    title: {
        fontSize: 20,
    },
})
