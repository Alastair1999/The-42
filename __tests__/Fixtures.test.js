import React from 'react';
import TestRenderer from 'react-test-renderer';
import Fixtures from "../src/screens/Fixtures";
import PremiershipFixtures from "../../src/screens/PremiershipFixtures";

// this test ensures that the UI does not change unexpectedly

test('renders correctly', () => {
  const tree = TestRenderer.create(<Fixtures />).toJSON();
  expect(tree).toMatchSnapshot();
});

// these next 4 test ensures that the buttons for each of leagues work

it('should go to premiership fixtures page', () => {
  const navigation= {navigate: () => {}}
  spyOn(navigation, 'navigate')

  const page = render(<Fixtures navigation={navigation}/>)

  const premiershipOpacity = page.getByTestId('premiershipOpacity')

  fireEvent.press(premiershipOpacity);

  expect (navigation.navigate).toHaveBeenCalledWith("PremiershipFixtures");

});

it('should go to championship fixtures page', () => {
  const navigation= {navigate: () => {}}
  spyOn(navigation, 'navigate')

  const page = render(<Fixtures navigation={navigation}/>)

  const championshipOpacity = page.getByTestId('champtionshipOpacity')

  fireEvent.press(championshipOpacity);

  expect (navigation.navigate).toHaveBeenCalledWith("ChampionshipFixtures");

});

it('should go to league one fixtures page', () => {
  const navigation= {navigate: () => {}}
  spyOn(navigation, 'navigate')

  const page = render(<Fixtures navigation={navigation}/>)

  const leagueOneOpacity = page.getByTestId('leagueOneOpacity')

  fireEvent.press(leagueOneOpacity);

  expect (navigation.navigate).toHaveBeenCalledWith("LeagueOneFixtures");

});

it('should go to league two fixtures page', () => {
  const navigation= {navigate: () => {}}
  spyOn(navigation, 'navigate')

  const page = render(<Fixtures navigation={navigation}/>)

  const leagueTwoOpacity = page.getByTestId('leagueTwoOpacity')

  fireEvent.press(leagueTwoOpacity);

  expect (navigation.navigate).toHaveBeenCalledWith("LeagueTwoFixtures");

});
