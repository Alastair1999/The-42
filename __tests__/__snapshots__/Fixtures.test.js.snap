// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RNCSafeAreaView
  edges={
    Array [
      "bottom",
      "left",
      "right",
      "top",
    ]
  }
  style={
    Array [
      undefined,
      Object {
        "backgroundColor": "#f7f7f7",
        "flex": 1,
      },
    ]
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderBottomWidth": 1,
        "borderColor": "#c0c0c0",
        "flexDirection": "row",
        "height": 64,
        "justifyContent": "space-between",
      }
    }
  >
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "flex-start",
          "flex": 1,
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "flex-start",
            "flex": 1,
            "justifyContent": "center",
            "paddingHorizontal": 20,
          }
        }
      >
        <Text />
      </View>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flex": 4,
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "justifyContent": "center",
            "paddingHorizontal": 20,
          }
        }
      >
        <Text
          fontWeight="bold"
          size="lg"
          style={
            Object {
              "color": "#050505",
              "fontFamily": "Ubuntu_700Bold",
              "fontSize": 16,
            }
          }
        >
          Fixtures
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "flex-end",
          "flex": 1,
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "flex-end",
            "flex": 1,
            "justifyContent": "center",
            "paddingHorizontal": 20,
          }
        }
      />
    </View>
  </View>
  <RCTScrollView
    contentContainerStyle={true}
    style={
      Object {
        "flex": 1,
        "flexDirection": "column",
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fff",
            "flex": 1,
          }
        }
      >
        <Text
          style={
            Object {
              "fontSize": 20,
              "paddingBottom": 15,
              "paddingTop": 15,
            }
          }
        >
           Select a league to view fixtures
        </Text>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-evenly",
              "padding": 40,
            }
          }
        >
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "red",
                "borderRadius": 10,
                "flexDirection": "row",
                "height": 60,
                "justifyContent": "center",
                "opacity": 1,
                "width": 250,
              }
            }
          >
            <Text
              style={
                Object {
                  "fontSize": 20,
                  "textAlign": "center",
                  "textTransform": "uppercase",
                }
              }
            >
              Cinch Premiership
            </Text>
            <Image
              resizeMode="contain"
              source={1}
              style={
                Object {
                  "height": 38,
                  "width": 35,
                }
              }
            />
          </View>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-evenly",
              "padding": 40,
            }
          }
        >
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "red",
                "borderRadius": 10,
                "flexDirection": "row",
                "height": 60,
                "justifyContent": "center",
                "opacity": 1,
                "width": 250,
              }
            }
          >
            <Text
              style={
                Object {
                  "fontSize": 20,
                  "textAlign": "center",
                  "textTransform": "uppercase",
                }
              }
            >
              Cinch Championship
            </Text>
            <Image
              resizeMode="contain"
              source={1}
              style={
                Object {
                  "height": 38,
                  "width": 35,
                }
              }
            />
          </View>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-evenly",
              "padding": 40,
            }
          }
        >
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "red",
                "borderRadius": 10,
                "flexDirection": "row",
                "height": 60,
                "justifyContent": "center",
                "opacity": 1,
                "width": 250,
              }
            }
          >
            <Text
              style={
                Object {
                  "fontSize": 20,
                  "textAlign": "center",
                  "textTransform": "uppercase",
                }
              }
            >
              Cinch League One
            </Text>
            <Image
              resizeMode="contain"
              source={1}
              style={
                Object {
                  "height": 38,
                  "width": 35,
                }
              }
            />
          </View>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-evenly",
              "padding": 40,
            }
          }
        >
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "red",
                "borderRadius": 10,
                "flexDirection": "row",
                "height": 60,
                "justifyContent": "center",
                "opacity": 1,
                "width": 250,
              }
            }
          >
            <Text
              style={
                Object {
                  "fontSize": 20,
                  "textAlign": "center",
                  "textTransform": "uppercase",
                }
              }
            >
              Cinch League Two
            </Text>
            <Image
              resizeMode="contain"
              source={1}
              style={
                Object {
                  "height": 38,
                  "width": 35,
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</RNCSafeAreaView>
`;
